vimms/Controller/misc.py

```diff
@@ -49,7 +49,7 @@ def set_tasks(self, schedule):
         self.initial_task = schedule[0]  # used for sending the first scan
         self.tasks = schedule[1:]  # used for sending all the other scans
 
-    def handle_scan(self, scan, outgoing_queue_size, pending_tasks_size):
+    def handle_scan(self, scan, current_size, pending_size):
```

Comment:
Are current_size and pending_size needed by the contollers?

Reply:
Not at the moment. It was passed to controller before because we thought the controller might want to know these sizes in order to do .. something with them.